hcl/hcl.go

```diff
@@ -1231,16 +1361,18 @@ func (p *TerramateParser) parseTerramateSchema() (Config, error) {
 
 	logger.Trace().Msg("Range over unmerged blocks.")
 
-	var foundstack bool
-	var stackblock *ast.Block
+	var foundstack, foundVendor bool
+	var stackblock, vendorBlock *ast.Block
+
 	for _, block := range rawconfig.UnmergedBlocks {
 		// unmerged blocks
 
 		logger := logger.With().
 			Str("block", block.Type).
 			Logger()
 
-		if block.Type == StackBlockType {
+		switch block.Type {
+		case StackBlockType:
```

Comment:
I'm wondering now why we have this just for "stack" haha I think I added this but don't remember haha

Reply:
We use this exported constant on other parts of the code to avoid duplicating the knowledge of what is the stack block name. We don't have this problem with other blocks I think.